backend/src/main/java/com/daedan/festabook/place/domain/PlaceDetail.java

```diff
@@ -77,4 +91,81 @@ public PlaceDetail(
                 endTime
         );
     }
+
+    private void validatePlace(Place place) {
+        if (place == null) {
+            throw new BusinessException("플레이스는 null일 수 없습니다.", HttpStatus.BAD_REQUEST);
+        }
+    }
+
+    private void validateTitle(String title) {
+        if (title == null) {
+            return;
+        }
+
+        if (title.trim().isEmpty()) {
+            throw new BusinessException("플레이스의 이름은 공백이 될 수 없습니다.", HttpStatus.BAD_REQUEST);
```

Comment:
```suggestion
            throw new BusinessException("플레이스의 이름은 공백일 수 없습니다.", HttpStatus.BAD_REQUEST);
```

뭔가 대부분 수동 느낌이 어색한 것 같아요...